04/app.js

```diff
@@ -1,5 +1,44 @@
-document.addEventListener('DOMContentLoaded', init);
+const key = '3f8b882da33d4731a149cd2cb80b54e4';
+const lang = 'pl';
+const path = `https://api.weatherbit.io/v2.0/current?key=${key}&lat=${lat}&lon=${long}&lang=${lang}`;
```

Comment:
Nie mamy zmiennych `lat`, `long` więc nie ma szans aby to działało.
Gdybyś `path` wrzucała do wnętrza getLocation to już prędzej ;)